css/Drawers.less

```diff
@@ -64,6 +64,7 @@
 	top: 0;
 	left: 0;
 	right: 0;
+	z-index: 1;
```

Comment:
I would suggest reverse the order of component block instead of change z-index here.
You can switch the order of handleContainer and drawers.